framework/integration/wrapper/spec.go

```diff
@@ -0,0 +1,20 @@
+// +build k8srequired
+
+package wrapper
+
+import "github.com/giantswarm/operatorkit/framework"
+
+type Interface interface {
+	// CRUD functions for objects.
+	CreateObject(namespace string, obj interface{}) (interface{}, error)
+	DeleteObject(name, namespace string) error
+	GetObject(name, namespace string) (interface{}, error)
+	UpdateObject(namespace string, obj interface{}) (interface{}, error)
+
+	// Functions for test setup and teardown.
+	MustSetup(namespace string)
+	MustTeardown(namespace string)
+
+	// Getters and setters.
+	Framework() *framework.Framework
+}
```

Comment:
Why is it a wrapper? What does it wrap?

Reply:
Naming is hard. 
Wrapper made sense to me because we wrap operations and the framework setup here. Do you have a better idea for a name?

Reply:
Ack. I saw ppl calling things like that `TestHarness` :)